src/desktop/components/artist_page_cta/view.coffee

```diff
@@ -35,12 +35,14 @@ module.exports = class ArtistPageCTAView extends Backbone.View
     @alreadyDismissed = false
     @signupIntent = 'Artist CTA Banner'
     @afterAuthPath = window.location
-    @$window.on 'scroll', _.throttle(@maybeShowOverlay, 200)
+    @$window.on 'scroll', _.once(@maybeShowOverlay)
     mediator.on 'clickFollowButton', @fullScreenOverlay
     mediator.on 'clickHeaderAuth', @fullScreenOverlay
+    mediator.on("artist_cta:account_creation", (options) => analytics.track("Created Account", options))
 
-  maybeShowOverlay: (e) =>
-    @fullScreenOverlay(e) if @$window.scrollTop() > @desiredScrollPosition and not @alreadyDismissed
+  maybeShowOverlay: () =>
```

Comment:
This seems to change the behavior away from scrolling down to twice the current viewport size to launch the full screen CTA, and to a 4 second delay (irregardless of any scrolling).

Seems very intentional to me :) Just making sure since that is a behavior change (vs. simply adding analytics).

Reply:
Yes, that's intentional. We want to be able to match the trigger action to be consistent with how we instrument the new modals which is based on scroll + delay.